### ISSUE TITLE
_lp_git_head_status shows an error if there's an error during interactive rebase

### ISSUE THREAD

**Issue by todorowww** [nojhan/liquidprompt#758]
Shell & Operating system:
- macOS 13.0.1 (zsh 5.8.1)
- Ubuntu 22.04 (bash 5.1)

Liquid Prompt version: v2.1.2
master branch commit d02c669466208d16539d6f8030dcc7c13a711dcc

### Steps to Reproduce
Start an interactive rebase
Make a typo in the editor when writing commands. I made a typo of `quesh` instead of `squash`.
Save and exit editor. You will notice an error.

### Expected Behavior
Don't display an error, just display the prompt in the state of REBASE-i

### Current Behavior
```
error: invalid line 2: quesh de669a7 Update f1(1)
You can fix this with 'git rebase --edit-todo' and then run 'git rebase --continue'.
Or you can abort the rebase with 'git rebase --abort'.
_lp_git_head_status:6: no such file or directory: PROJECT_PATH/.git/rebase-merge/msgnum
_lp_git_head_status:7: no such file or directory: PROJECT_PATH/.git/rebase-merge/end
[:PROJECT_PATH] aed92c8 REBASE-i 12s 1 ±
```

### Possible Solution
Before reading, we should test if the file exists prior to reading from it.

**Comment by Rycieos** [comment]
Good debugging steps, can confirm.

**Comment by Rycieos** [comment]
Resolved with 0126fb3
